doc/api_2_0_spec/api_2_0_spec.md

```diff
@@ -771,6 +771,33 @@ the returned data would look like:
 }
 ```
 
+## api/v2/stories_public/is_syndicated_ap (POST)
+
+Detect whether a given block of content is likely to be ap syndicated content by looking for certain signals in the text
```

Comment:
Maybe have nicer casing, i.e. "AP" instead of "ap"? When it's in lowercase here, it's ambiguous on whether it's AP the abbreviation or it's a typo for something else. IMHO uppercase / lowercase letters look more professional, esp. for abbreviations (AP, HTML, JSON, ...).

Reply:
left as is.  I prefer the lower case.